docs/docs/guides/installation/run/docker-compose.yaml

```diff
@@ -16,11 +16,28 @@ services:
     ports:
       - '8080:8080'
 
-  db:
+  pg:
     restart: 'always'
     networks:
       - 'zitadel'
-    image: 'cockroachdb/cockroach:v22.1.0'
+    image: postgres:14.4
```

Comment:
maybe we should provide a postgres as well, but default to crdb

Reply:
@elio might have an idea how or if we should create a guide for postgres